src/OrchardCore/OrchardCore.Queries.Abstractions/OrchardCore.Queries.Abstractions.csproj

```diff
@@ -4,14 +4,20 @@
     <RootNamespace>OrchardCore.Queries</RootNamespace>
     <!-- NuGet properties-->
     <Title>OrchardCore Queries Abstractions</Title>
-    <Description>$(OCCMSDescription)
+    <Description>
+      $(OCCMSDescription)
```

Comment:
These line breaks will show up on nuget.org too, so having an empty line before and after the package description. Please revert.

Reply:
damn VS! it auto format the csproj file. I'll fix

Reply:
reverted